model/base_model.py

```diff
@@ -10,9 +10,12 @@ class BaseModel:
     This abstract class will take care of all common code that is to be shared across all model variants.
     """
 
-    def __init__(self, name, problem="rproblem2"):
+    def __init__(self, name, problem="rproblem2", mip_focus='default', solution_limit='default'):
```

Comment:
Are these default values correct? Gurobi usually uses integer arguments, so this seems a little weird.

Reply:
This should, based on the documentation on setParam, be the best way to set the parameters to default values. I think 'default' gives more meaning than writing mip_focus=1 and solution_limit=MAXINT.

Reply:
Great! I agree that `default` makes more sense!